.github/workflows/push-images.yml

```diff
@@ -0,0 +1,46 @@
+name: "Push images to Docker Hub"
+on: workflow_dispatch
+jobs:
+  build:
+    name: "Builds images and push them to Docker Hub"
+    runs-on: "ubuntu-22.04"
+    steps:
+      - name: "Check out repository"
+        uses: "actions/checkout@v4"
+      - name: "Set up buildx"
+        uses: "docker/setup-buildx-action@v3"
+        with:
+          version: latest
+          driver-opts: image=moby/buildkit:v0.13.0
```

Comment:
Because the environment in which this is executed uses by default buildkit `v0.12` and we need `v0.13`+ in order for the multitarget feature to work. So we have to directly specify the version.